docs/targeted-tutorials/read-local-data-in-k8s-silo.md

```diff
@@ -0,0 +1,78 @@
+# Read local data in an on-premises Kubernetes silo
+
+## Background
+In many Federated Learning (FL) applications, the silos (compute + user data) need to be located on-premises. Azure ML can accommodate such external silos, provided the computes are running a Kubernetes (k8s) cluster. When the user data are physically located on the same machine that holds the k8s cluster, the Azure ML job running in the k8s cluster needs access to the local file system. 
+
+## Objective and contents
+This tutorial will show you how to access, within an Azure ML job running on an on-premises k8s cluster, some data in the local file system. First, we will explain how to create the k8s cluster using [kind](https://kind.sigs.k8s.io/), and how to configure it properly with [Persistent Volumes](https://kubernetes.io/docs/concepts/storage/persistent-volumes/). Then, we will create an external silo based on this k8s cluster. Finally, we will run a simple test job to prove that an Azure ML job running on the k8s cluster can indeed access the local file system.
+
+> **:grey_exclamation: Note**:
+> To mimic _in the cloud_ an on-premises silo, one can create the k8s cluster in an Azure VM instead of a local machine. The steps described in this tutorial still apply.
+
+## Prerequisites
+For creating the k8s cluster:
+- a machine with mock data located at `/path/to/data/data_file.txt`;
+- [kind](https://kind.sigs.k8s.io/) with version >= 0.16.0 (the current tutorial was made using that version)
+
+For creating an external silo based on the k8s cluster:
+- see the prerequisites listed [over there](../provisioning/external-silos.md).
+
+## Procedure
+
+### 1. Provision and configure your local k8s cluster
```

Comment:
there is some overlap between this and the external-silo tutorial, is it possible to merge them together?

Reply:
@jfomhover I'll look into what exactly it would take to merge, if you feel strongly about this. I do favor keeping separate though, here's my thinking.

I initially chose to keep those separate because the scopes are quite different. The external-silos tutorial is not only about on-prem silos, but also about Azure silos in a different tenant. This second type of silos will not need to read local data. Furthermore, not every user for on-prem silos will be interested in reading local data (they might have their own data server on their network that they can pull from). 

Last thing, merging those 2 big docs would result in an even bigger doc. That bigger doc would cover several scenarios, somewhat related but not quite the same. The user would then have to look for the relevant parts in that long doc. I think the experience of having distinct locations for info corresponding to distinct scenarios is better, even if it comes at the cost of a few duplicated instructions.